charts/alluxio/templates/master/statefulset.yaml

```diff
@@ -173,11 +173,25 @@ spec:
           {{- if .Values.master.resources  }}
 {{ include "alluxio.master.resources" . | indent 10 }}
           {{- end }}
+          {{- if eq .Values.master.mountConfigStorage "configmap"}}
+          command: ["/etc/fluid/scripts/entrypoint.sh"]
+          {{ else }}
           command: ["/entrypoint.sh"]
+          {{- end }}
           {{- if .Values.master.args }}
           args:
 {{ toYaml .Values.master.args | trim | indent 12 }}
           {{- end }}
+          {{- if eq .Values.master.mountConfigStorage "configmap"}}
+          startupProbe:
+            exec:
+              command:
+                - "/etc/fluid/scripts/mount.sh"
```

Comment:
I suggest adding `>> /proc/1/fd/1` to help the user know what happened when debgging.

Reply:
ok, fixed.